pylsp/__main__.py

```diff
@@ -57,13 +58,22 @@ def add_arguments(parser):
         help="Increase verbosity of log output, overrides log config file"
     )
 
+    parser.add_argument(
+        '-V', '--version', action='store_true',
```

Comment:
Is it common to use the `-V` option for showing the version? I don't think I've ever seen this used before. And `-v` is usually used for verbose logging so it seems a little to close to that. Is having `--version` not enough?

Reply:
I copied it from `python`, but I can remove the short option if needed
```python --help
usage: python [option] ... [-c cmd | -m mod | file | -] [arg] ...
-v     : verbose (trace import statements); also PYTHONVERBOSE=x
         can be supplied multiple times to increase verbosity
-V     : print the Python version number and exit (also --version)
```

Reply:
Also, it looks like argparse have native support for "--version" handling. See:

![Screenshot 2021-06-24 at 09 06 19](https://user-images.githubusercontent.com/153197/123218031-83d29e00-d4cb-11eb-9bdf-c7de84956ed9.png)

https://docs.python.org/3/library/argparse.html

Reply:
Hm.  That would simplify the patch a bit.

Reply:
If python does it then I guess it's fine.

But could use that "native" way to add the argument.

Reply:
OK. I've pushed a new version.  Thanks